packages/e2e_tests/cypress/e2e/exam-timetable-register-as-myself.cy.js

```diff
@@ -0,0 +1,82 @@
+import { PO_ExaminationTimetable } from '../pageObject/Examination-TimeTable/PO_ExaminationTimetable';
+import { PO_ProjectSearch } from '../pageObject/Search-and-project-pages/PO_ProjectSearch';
+import { PO_ProjectPage } from '../pageObject/Search-and-project-pages/PO_ProjectPage';
+
+const examinationTimetable = new PO_ExaminationTimetable();
+const projectSearch = new PO_ProjectSearch();
+const projectPage = new PO_ProjectPage();
+
+describe('User registers as themselves to have their say against the examination timetable', () => {
+	it('Navigates to the examination timetable page for a project and start the journey', () => {
+		cy.clearCookies();
+		cy.visit('/projects/EN010120/examination-timetable');
```

Comment:
This test assumes EN010120 will be at stage Examination and have an open deadline in any env we run it.
Similar for WA020002 being in registration.  

Something to think about for the future, we could introduce a variable to the test EXAMINATION_CASE_REFERENCE.

Reply:
Will invetigate how we make configurable in future work